variables.tf

```diff
@@ -40,7 +40,7 @@ variable "region" {
 variable "member_memory_mb" {
   type        = number
   description = "Allocated memory per-member. See the following doc for supported values: https://cloud.ibm.com/docs/databases-for-mysql?topic=databases-for-mysql-resources-scaling"
-  default     = 1024
+  default     = 4096
```

Comment:
```hcl
╷
│ Error: ---
│ id: terraform-23dab597
│ summary: "1 error occurred:\n\t* member group memory must be >= 4096 and <= 114688
│   in increments of 128\n\n"
│ severity: error
│ resource: ibm_database
│ operation: CustomizeDiff
│ component:
│   name: github.com/IBM-Cloud/terraform-provider-ibm
│   version: 1.67.1
│ ---
│ 
│ 
│   with module.mysql_db.ibm_database.mysql_db,
│   on ../../main.tf line 55, in resource "ibm_database" "mysql_db":
│   55: resource "ibm_database" "mysql_db" {
│ 
```

I think technically 4G is not enforced on the deprecated (host_flavor = null) yet. However, instances are, or will shortly be updated automatically. It is unclear exactly what effect this will have under terraform, where the deployed state is potentially different to the configured state.
It is enforced on multitenant  (error above) and not applicable on Isolated (host_flavour=machine type).

Do you want to keep 1G and put an error out if multitenant is selected with less than 4G? Or keep 4G which is the minimum for the new model that is being encouraged/automatically updated.

Reply:
I think 4G is fine as the default then. Potentially just add a comment in the code on why this is set as such - eg: summary of what you mention to make it easier in the future.

Reply:
I merged it for now - but we can track adding the comment in a separate issue